vonk/releasenotes/releasenotes.rst

```diff
@@ -10,8 +10,64 @@ Release notes Vonk
    releasenotes_old
    security_notes
 
+.. _vonk_releasenotes_300-beta2:
+
+Release 3.0.0-beta2
+--------------------
+
+.. attention::
+
+   We updated the :ref:`vonk_securitynotes`.
+
+Database
+^^^^^^^^
+
+Note the changes in :ref:`3.0.0-beta1 <vonk_releasenotes_300beta1>`, but there are no new changes in beta2.
+
+Feature
+^^^^^^^
+
+#. Subscription works for R4 also. Note that a Subscription will only be activated for resource changes in the same FHIR version as the Subscription itself.
+#. Administration Import of files and directories works for R4 also. Use a directoryname that ends with ``.R4`` for R4 conformance resources.
```

Comment:
I think this could be clearer. Something similar to the text in conformanceresources.rst

Reply:
Added links for both Subscriptions and Admin import